### PR TITLE
    Add copts for SwiftLibrary

### PR DESCRIPTION
**Pull request by jerrymarino**
Add copts for SwiftLibrary

This PR adds default copts for SwiftLibrary. Additionally, it cleans up
a few of the methods used to build out copts from xcconfig

### CODE REVIEW

File: Sources/PodToBUILD/ObjcLibrary.swift [hunk 1]
```diff
@@ -388,14 +381,22 @@ public struct ObjcLibrary: BazelTarget, UserConfigurable, SourceExcludable {
         let extraDepNames = extraDeps.map { bazelLabel(fromString: ":\($0)") }
         deps = AttrSet(basic: extraDepNames) <> mpPodSpecDeps
 
+        // Adds minimal, non specified Xcode defaults
         let extraCopts: AttrSet<[String]>
         if case .cpp = sourceType {
```
Comment (jparise) [inline Sources/PodToBUILD/ObjcLibrary.swift:386]: It's too bad Swift's `case`'s aren't expressions.

File: Sources/PodToBUILD/SwiftLibrary.swift [hunk 2]
```diff
@@ -150,7 +150,23 @@ public struct SwiftLibrary: BazelTarget {
...
+            .compilerFlags(for: fallbackSpec)
+
+        let includes = objcFlags.filter { $0.hasPrefix("-I") }
+
+        // Insert the clang import flags
+        let clangImporterCopts = (includes + ["-DCOCOAPODS=1"])
+            .reduce([String]()) { $0 + ["-Xcc", $1] }
+        self.copts = AttrSet(basic: [
+            "-DCOCOAPODS",
+        ] + swiftFlags + clangImporterCopts)
```
Comment (jparise) [inline Sources/PodToBUILD/SwiftLibrary.swift:168]: This adds both `-DCOCOAPODS=1` and `-DCOCOAPODS`?

Comment (jerrymarino) [reply Sources/PodToBUILD/SwiftLibrary.swift:168]: This adds `-DCOCOAPODS` and `-DCOCOAPODS=1` to the clang importer ( via `-Xcc` ) - by doing this, clang sees `-DCOCOAPODS=1`. I'm not 100% sure why it doesn't pass `-DCOCOAPODS` to swift, but this is the behavior in Xcode so I wanted to make it identical

File: Sources/PodToBUILD/ObjcLibrary.swift (hunk 1 above)
Comment (jerrymarino) [reply Sources/PodToBUILD/ObjcLibrary.swift]: I think they might have added this in Swift 5. 🥳

**Pull request merged**